modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -23,12 +24,10 @@ This procedure requires knowledge of the workload utilizing the kernel module an
 
 * Ensure that the user workload (the application running in the cluster that is accessing kernel module) is not running on the node prior to kernel module unloading and that the workload is back running on the node after the new kernel module version has been loaded.
 
-* Ensure that the device plugin managed by KMM on the node is unloaded before the upgrade process and reloaded following the upgrade process.
-
-* Due to Kubernetes limitations in label names, the combined length of `Module` name and namespace must not exceed 39 characters.
-
 .Procedure
 
+. Ensure that the device plugin managed by KMM on the node is unloaded.
```

Comment:
@cdvultur This step and line 66 was previously in the Prerequisites section. Please ok this if the steps are in the right order.